app/views/editor/categories/_category.html.haml

```diff
@@ -0,0 +1,11 @@
+%tr{class: "depth#{category.depth}"}
+  %td= category.name
+  %td= category.description
+  %td= category.slug
+  %td= link_to 'Show', editor_category_url(category)
+  %td= link_to 'Edit', edit_editor_category_url(category)
+  %td= link_to 'Destroy', editor_category_url(category), method: :delete, data: { confirm: 'Are you sure?' }
+  %td= link_to_unless category.first?, '▲', editor_category_order_url(category_id: category, move_to: "left", target: category.higher_item&.id), method: :patch
```

Comment:
> PUT to a URL creates/replaces the resource in its entirety at the client defined URL.
> PATCH to a URL updates part of the resource at that client defined URL.

ref: http://stackoverflow.com/questions/630453/put-vs-post-in-rest

In this case, these request parameters are the part of resource or itself?